backends/tfhe-cuda-backend/cuda/src/integer/scalar_bitops.cuh

```diff
@@ -36,7 +36,7 @@ __host__ void host_integer_radix_scalar_bitop_kb(
         stream, lwe_array_out, lwe_array_input, bsk, ksk, num_clear_blocks,
         lut);
 
-    if (op == SCALAR_BITAND && num_clear_blocks < num_radix_blocks) {
+    if (op == SCALAR_BITAND) {
```

Comment:
I think we still need the second check. If `num_clear_blocks > num_radix_blocks` we would have trouble.

Reply:
An alternative is to modify the memory functions to handle negative inputs.

Reply:
You're right I'll put it back :+1: